simpa/core/simulation_modules/acoustic_forward_module/simulate_2D.m

```diff
@@ -147,11 +147,19 @@
     datacast = 'single';
 end
 
+
```

Comment:
It would be super helpful if you would add a comment like `Extracts the file name from the optical path, adds the current time in milliseconds and the process id to make the file name unique` so that this passage is directly understandable